hypertrace-graphql-metric-schema/src/main/java/org/hypertrace/graphql/metric/schema/MetricAggregationQueryable.java

```diff
@@ -11,6 +16,13 @@
   @GraphQLField
   @GraphQLName(METRIC_FIELD_NAME)
   @GraphQLNonNull
-  BaselinedMetricAggregationContainer metric(
-      @GraphQLName(MetricKeyArgument.ARGUMENT_NAME) @GraphQLNonNull String key);
+  default BaselinedMetricAggregationContainer metric(
+      @GraphQLName(MetricKeyArgument.ARGUMENT_NAME) @Nullable String key,
```

Comment:
will this `@Nullable String key` be deprecated later?

Reply:
yep, exactly